src/main/conf/ccm-fetch.cron

```diff
@@ -1,5 +1,4 @@
-# run regularly ccm-fetch. To avoid lost notifications
+# To avoid lost notifications, run ccm-fetch regularly.
 #
-
-0 * * * * root (perl -e 'sleep rand 3600'; /usr/sbin/ccm-fetch) >>/var/log/ccm-fetch.log 2>&1
-
+RANDOM_DELAY=60
+0 * * * * root (date --iso-8601=seconds --utc; /usr/sbin/ccm-fetch) >>/var/log/ccm-fetch.log 2>&1
```

Comment:
with high enough debug, lines like https://github.com/quattor/CCM/blob/df0e82674d0643f12b6700aa1e69c72f7f75e9c3/src/main/perl/Fetch/ProfileCache.pm#L436 report actual data. if you have `world_readable`, this is no issue.
we could also log to a `/var/log/ccm/ccm-fetch.log` with 644, and people can apply permissions with `dirperm` on the whole logdir. (setting the permissions/ownership directly is not going to be that easy anyway)